apps/test/unit/templates/teacherDashboard/SectionActionDropdownTest.js

```diff
@@ -150,7 +150,7 @@ describe('SectionActionDropdown', () => {
       <SectionActionDropdown
         {...DEFAULT_PROPS}
         sectionData={sections[3]}
-        userId={1539340}
+        userId={90}
```

Comment:
I would recommend not changing this since it's the same either value (right?)

Reply:
I was trying to follow up on the comment you left on [Kaitie's PR](https://github.com/code-dot-org/code-dot-org/pull/51825) implementing this ('thoughts on using 90 for magic number consistency? Or is this one different?') to make them all the same again for clarity. I recognize that it would've been ideally in its own followup before this pr. Thoughts on updating it now?

Reply:
Ok yes I see, do it!